src/mongo/client/dbclient.cpp

```diff
@@ -180,6 +188,24 @@ namespace mongo {
             }
             return bob.obj();
         }
+
+        string _authKeyCopyDBMongoCR(const string& username,
```

Comment:
you don't need the preceding underscore in the declaration anymore since this isn't a private method.